packages/fluent/src/FluentTheme.ts

```diff
@@ -53,7 +53,7 @@ export class FluentTheme extends ThemeImpl implements IThematicFluentTheme {
 		return this._fTheme.semanticColors
 	}
 	get rtl() {
-		return this._fTheme.rtl
+		return this._fTheme.rtl ?? false
```

Comment:
we are implementing as passthrough, which can be undefined

Reply:
Update essex tsconfig remove this strictness check